cmd/db/schema_migration_manager.go

```diff
@@ -0,0 +1,122 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/stellar/go/support/log"
+
+	"github.com/stellar/stellar-disbursement-platform-backend/db"
+	"github.com/stellar/stellar-disbursement-platform-backend/db/migrations"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/utils"
+)
+
+type SchemaMigrationManager struct {
+	MigrationRouter        migrations.MigrationRouter
+	SchemaName             string
+	SchemaDatabaseDNS      string
```

Comment:
```suggestion
	SchemaDatabaseDSN     string
```

Reply:
Addressed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/265/commits/b51454d9bb175b8579a04ef3adec7347cd489980.